src/main/scala/com/moesif/filter/MoesifApiFilter.scala

```diff
@@ -301,3 +260,24 @@ trait MoesifApiFilterComponents {
   lazy val moesifApiFilterConfig: MoesifApiFilterConfig = MoesifApiFilterConfig.fromConfiguration(configuration)
   lazy val moesifApiFilter: MoesifApiFilter             = new MoesifApiFilter(moesifApiFilterConfig)(materializer)
 }
+
+object MoesifApiFilter{
+  def buildUriHelper(host: String, uri: String, secure: Boolean): String = {
+    if (uri.contains("://")) {
+      uri
+    }
+    else {
+      val protocol = if (secure) "https" else "http"
```

Comment:
This could be https:// or http://
It will reduce one less string mutation below

Reply:
Changed